statuscake/resource_statuscaketest.go

```diff
@@ -175,10 +185,6 @@ func resourceStatusCakeTest() *schema.Resource {
 			"status_codes": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Default: "204, 205, 206, 303, 400, 401, 403, 404, 405, 406, " +
```

Comment:
These codes will have default values on statuscake side, right?

Reply:
Yes. I only verified 502 since it’s the easiest way to force a failure, but they do appear to have some set of default failure codes.